js/client-chat.js

```diff
@@ -1266,9 +1266,19 @@
 					break;
 
 				case 'uhtml':
-					this.$chat.append('<div class="notice uhtml-' + toId(row[1]) + '">' + Tools.sanitizeHTML(row.slice(2).join('|')) + '</div>');
+					var $elements = this.$chat.find('div.uhtml-' + toId(row[1]));
+					if (!$elements.length) {
+						this.$chat.append('<div class="notice uhtml-' + toId(row[1]) + '">' + Tools.sanitizeHTML(row.slice(2).join('|')) + '</div>');
+					} else {
+						if (!toId(row.slice(2).join('|'))) {
```

Comment:
`else if`

Heck, this should probably be the first `if`, the appending should be the `else if`.